index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+
+
+<body>
+    <main class="top-section">
+        <section class="header">
+            <div class="app-bar">
+                <ul>
+                    <li class="app-bar-item1"><img src="/images/icons/time.png" alt="Time"></li>
+                    <li class="app-bar-item2"><img src="/images/icons/Group4.png" alt="Phone Status"></li>
+                </ul>
+            </div>
+            <nav class="menu-icon">
+                <ul>
+                    <li class="menu-item1"><img src="/images/icons/logo.png" alt="Logo"></li>
+                    <li class="menu-item2"><img src="/images/icons/Union.png" alt="Main menu"></li>
+                </ul>
+            </nav>
+        </section>
+        <section class="headline">
+            <div class="headline-title">
+                Hey There<br />
+                I'm Eric.
+            </div>
```

Comment:
The main title of a website is a `h1` tag for search engine optimization you can check this  [article](https://www.reliablesoft.net/h1-tag/) 😃